flask/boxwise_flask/models/base.py

```diff
@@ -1,12 +1,48 @@
-from peewee import CharField, IntegerField
+from datetime import datetime
 
-from ..db import db
+from boxwise_flask.db import db
+from boxwise_flask.models.organisation import Organisation
+from boxwise_flask.models.user import User
+from peewee import SQL, CharField, DateTimeField, ForeignKeyField, IntegerField
 
 
 class Base(db.Model):
-    organisation_id = IntegerField()
-    name = CharField()
-    currency_name = CharField(column_name="currencyname")
+    name = CharField(null=True)
+    currency_name = CharField(
+        column_name="currencyname", constraints=[SQL("DEFAULT 'Tokens'")]
```

Comment:
a lot of this is autogen from peewiz so its just straight from the db. I figured I'd try not to update the auto generated code too much in case we update the tables and have to gen things again.